src/Raven.Server/Rachis/FollowerAmbassador.cs

```diff
@@ -204,6 +204,16 @@ private unsafe void Run()
                             }
                         }
                     }
+                    catch (TopologyMismatchException e)
+                    {
```

Comment:
This is a _really_ dangerous operation, to automatically remove a node from the cluster like that.
Better to have it continue to fail. 

And note that such operations _have_ to be `Operations` logged and alerted, if we allowed that, which we won't do.